### ISSUE TITLE
pachctl list-job foo/<commitid> broken

### ISSUE THREAD

**Issue by JoeyZwicker** [pachyderm/pachyderm#2640]
These pachctl commands are broken

```
# return all jobs whose input commits include foo/XXX and bar/YYY
$ pachctl list-job foo/XXX bar/YYY

# return all jobs in pipeline foo and whose input commits include bar/YYY
$ pachctl list-job -p foo bar/YYY
```

```
$ pc list-commit images
REPO                ID                                 PARENT                             STARTED             DURATION             SIZE
images              2f664f9f8b4b4a4aa5e9bcf0a95ec167   2d1c931a0ad64494b2ff9a58dfd5ffad   22 minutes ago      Less than a second   1.623MiB
images              2d1c931a0ad64494b2ff9a58dfd5ffad   b7fad4b7fdc74ae492bb091f7c6ec6f3   About an hour ago   Less than a second   1.567MiB
images              b7fad4b7fdc74ae492bb091f7c6ec6f3   0e23aa73e35c4c49ba34dc0aad2fe83a   26 hours ago        Less than a second   1.562MiB
images              0e23aa73e35c4c49ba34dc0aad2fe83a   4016939e536541928eb426f8ee31f341   26 hours ago        Less than a second   1.474MiB
images              4016939e536541928eb426f8ee31f341   <none>                             26 hours ago        Less than a second   126.8KiB
Joseph-Zwickers-MacBook-Pro:~ joeyzwicker$ pc list-job images/2f664f9f8b4b4a4aa5e9bcf0a95ec167
expected 0 arguments, got 1
```

**Comment by sjezewski** [comment]
Hit this during a customer install. The workaround was hacky :-(

**Comment by jdoliner** [comment]
This was fixed in #2640.

Apparently adding "Fixes #XXXX" doesn't close #XXXX on merge like I thought it did.

**Comment by jdoliner** [comment]
This will be available in 1.7.

**Comment by sjezewski** [comment]
Ok :+1: 

Huh, I thought that worked too. Maybe its lower case `fixes` ?
